packages/components/docs/sass.md

```diff
@@ -3665,6 +3673,7 @@ Define theme variables from a map of tokens
   $text-02: map-get($theme, 'text-02') !global;
   $text-03: map-get($theme, 'text-03') !global;
   $text-04: map-get($theme, 'text-04') !global;
+  $text-05: map-get($theme, 'text-05') !global;
```

Comment:
Looks like ci is failing bc there's a reference to `$text-05` (auto generated in this sass doc) which is a new token. This is coming from of the generated `chore()` commits.

Reply:
Should these be excluded from the PR?

Reply:
commit: https://github.com/carbon-design-system/carbon/pull/4006/commits/174d7782277f46f3d83579e84a3314cac7489a3b

Reply:
It seems like `text-05` was added in: 36113ce679f5e851efe78292298b8f1acd814629 and synced in: https://github.com/carbon-design-system/carbon/commit/99f12f98e226b2ab1872b7379167d827e4fe7bd5

Does that match what you've found?

Reply:
yes